<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHough</class>
 <widget class="QDialog" name="DialogHough">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>theta</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="rhoSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSlider" name="thetaSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSlider" name="thresholdSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>threshold</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>rho</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="linesButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="circlesButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Circles</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
